speedcord/shard.py

```diff
@@ -68,6 +78,9 @@ async def close(self):
             await self.ws.close()
 
     async def read_loop(self):
+        """
+        Reads all messages and dispatches them to a gateway handler, which is defined in the client. 
```

Comment:
Maybe change this to `Receives data from the gateway and sends it to the correct place` or something similar?

Reply:
Something like `Receives data from the gateway and sends it to a handler` sounds about right

Reply:
Yep that sounds better